rcl_interfaces/msg/ParameterEventDescriptors.msg

```diff
@@ -2,6 +2,7 @@
 # It was an atomic update.
 # A specific parameter name can only be in one of the three sets.
 
+RclHeader header
```

Comment:
I would suggest using `rcl_header` here so as not to be confused with `Header header`.

Reply:
:+1: